scripts/test-fuzzed-MAC.py

```diff
@@ -47,12 +51,13 @@ def main():
         node = node.add_child(FinishedGenerator())
         node = node.add_child(ExpectChangeCipherSpec())
         node = node.add_child(ExpectFinished())
-        node = node.add_child(fuzz_mac(ApplicationDataGenerator(
-                                                        b"GET / HTTP/1.0\n\n"),
-                                       xors={pos:val}))
+        node = node.add_child( \
+            fuzz_seal(ApplicationDataGenerator( b"GET / HTTP/1.0\n\n"), xors={pos:val})
```

Comment:
The script as written now does seem to be working, I can see it connect with GCM to the server and getting bad records when it should.

fuzz_message() not used due to error TypeError: unsupported operand type(s) for ^=: 'str' and 'int' when it attempts to apply xor to a string. So I was assuming fuzz_message() wasn't written for this purpose.

I can try develop a fuzz_ciphertext() as you suggested if this is the best way forward.